src/style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-size: cover;
+  width: 12px;
+  height: 12px;
+  margin: 2px 6px 2px 2px;
+}
+
+.stars--0 > :nth-child(n + 1) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 > :nth-child(n + 2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 > :nth-child(n + 3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 > :nth-child(n + 4) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 > :nth-child(n + 5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 > :nth-child(n + 6) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
You can use coma to combine all selectors and don't repeat background-image property for each separate class